OMCompiler/Compiler/BackEnd/BackendEquation.mo

```diff
@@ -1648,8 +1648,12 @@ algorithm
         eqns = listAppend(reqn,eqns);
       then (eqn, (funcs, eqns));
 
-    else (inEq,inEqs);
-  end matchcontinue;
+    else
+      equation
+        true = Flags.isSet(Flags.FAILTRACE);
```

Comment:
Will it be OK if the fail trace is not set?

Reply:
It fails regardless if failtrace is set. It's only supposed to print a trace if the flag is set.